app/models/manageiq/providers/microsoft/infra_manager.rb

```diff
@@ -133,4 +155,13 @@ def build_connect_params(options)
 
     connect_params
   end
+
+  def vm_host_handle(vm, options)
+    require 'Scvmm/miq_scvmm_vm_ssa_info'
+    raise "no credentials defined in options #{options}" if self.missing_credentials?(options[:auth_type])
+    user = options[:user] || authentication_userid(options[:auth_type])
+    pass = options[:pass] || authentication_password(options[:auth_type])
+
+    MiqScvmmVmSSAInfo.new(vm.host.hostname, user, pass)
```

Comment:
And it turns out that hostname refers to the EMS which is the SCVMM, and we want the HyperV host here, which is vm.host.hostname, what I was originally using.  So I changed the first argument to MiqScvmmVmSSAInfo back to that.